apps/web/components/auth/layouts/AdminLayout.tsx

```diff
@@ -1,40 +0,0 @@
-"use client";
```

Comment:
- Now that we moved every settings page to app router, we don't need this. Instead we have `AdminLayoutAppDir` and `AdminLayoutAppDirClient` used in app router.
- I wanted to remove `SettingsLayout` too but it is used by `settings/admin/orgMigrations` pages. Can remove these pages? @hariombalhara